metric_learn/base_metric.py

```diff
@@ -288,7 +288,7 @@ def get_mahalanobis_matrix(self):
 
     Returns
     -------
-    M : `numpy.ndarray`, shape=(n_components, n_features)
+    M : `numpy.ndarray`, shape=(num_dims, n_features)
```

Comment:
shouldn't this always be `(n_features, n_features)`?

Reply:
that's right, I didn't pay attention, thanks